swiftide-indexing/src/transformers/prompts/metadata_qa_code.prompt.md

```diff
@@ -26,7 +26,13 @@ A1: It transforms strings into integers.
 Q2: What other internal parts does the code use?
 A2: A hasher to hash the strings.
 ```
+{% if context %}
 
+## Context
+```
+{{ context }}
+```
+{% endif %}
```

Comment:
I'd suggest to be a bit more specific here to the LLM, i.e. instead of `Context`, try `Outline of parent file`.

Reply:
ah in this place I think we actually want to be generic, so we can insert other things than just the outline as well.

Reply:
Nvm I'll change it to outline